### ISSUE TITLE
Home and Beacon separation

### ISSUE THREAD

**Issue by Turmi0** [sensorfu/sensorfu.github.io#16]
Is Home and Beacon described well enough as a separated components?

For example "Works with Raspberry Pi" means only Beacons but it is not mentioned in the text.

**Comment by eemil** [comment]
The entire page is fairly beacon-heavy. Until the "Get started" section, Home is only mentioned a few times and somewhat ambiguously ("Beacon calls Home"). This isn't necessarily a bad thing, since Beacon is the part of our product that the customer can see and feel, and what they will be deploying.

I think most readers will infer that anything written applies to Beacon, they might not even realize that Home is a component that they could deploy themselves.

**Comment by Turmi0** [comment]
Thats true.  On the other hand, price is currently based on number of Home -instances .
